index.html

```diff
@@ -41,8 +41,124 @@ <h3 class="link-1"> <a href="#"> LET'S CONNECT</a></h3>
                    </ul>
                 </div>
             </footer>
+        </section> 
+        </div>
+        <div class="work-title">My Work</div>
```

Comment:
- [ ] Good work @Ashe546, your title should use a header tag, kindly implement it.